src/main/java/core/basesyntax/service/operation/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.model.Fruit;
+
+public interface OperationHandler {
+    public abstract Fruit completeOperation(Fruit fruit);
```

Comment:
all interface methods is public and abstract by default 
```suggestion
    Fruit completeOperation(Fruit fruit);
```
also, it makes sense not to return `Fruit` but put the changed value directly in the storage.